nbs/15_gsa_gls/18-create_corr_mat_per_lv.ipynb

```diff
@@ -0,0 +1,865 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5ef66977",
+   "metadata": {
+    "papermill": {
+     "duration": 0.02266,
+     "end_time": "2022-07-19T14:46:13.019969",
+     "exception": false,
+     "start_time": "2022-07-19T14:46:12.997309",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Description"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "658d4f7c",
+   "metadata": {
+    "papermill": {
+     "duration": 0.008857,
+     "end_time": "2022-07-19T14:46:13.045657",
+     "exception": false,
+     "start_time": "2022-07-19T14:46:13.036800",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "**TODO UPDATE**\n",
```

Comment:
Again, do you want to do this now or leave it for a future PR?

Reply:
Yes, description added, thank you!